python/pythonmonkey/require.py

```diff
@@ -30,18 +30,33 @@
 from importlib import util
 
 from . import pythonmonkey as pm 
+node_modules = os.path.abspath(os.path.dirname(__file__) + "/../pminit/pythonmonkey/node_modules")
```

Comment:
I think this might be better suited as relative to the pminit package (cause the node_modules install is also relative to the pminit package). This should make it work in both the dev case and the end user case. Something like:
```suggestion
import pminit
node_modules = os.path.abspath(
  os.path.join(
    pminit.__path__[0],
    '..',
    'pythonmonkey',
    'node_modules'
  )
)
```

Reply:
Should we move `node_modules` into the `pminit` package dir instead?

Reply:
@Hamada-Distributed 
We should use `importlib` to avoid actually importing the `pmint`module. 
```suggestion
node_modules = os.path.abspath(
  os.path.join(
    importlib.util.find_spec("pminit").submodule_search_locations[0],
    '..',
    'pythonmonkey',
    'node_modules'
  )
)
```
https://docs.python.org/3/library/importlib.html#importlib.machinery.ModuleSpec.submodule_search_locations

Reply:
makes sense to me! I've approved your MR with those changes.

Reply:
This change was made in parallel on the main branch.  Now that I have merged this change in, nothing works any more. Can we please please please for the love of God run the tests before pushing to main?